index.js

```diff
@@ -309,14 +323,29 @@ export default class WorkspacesPlugin extends Plugin {
   }
 
   _buildReplacementDepencencyVersion(existingVersion, newVersion) {
-    let firstChar = existingVersion[0];
+    let prefix = '';
+    let firstChar = '';
+    let range = existingVersion;
+    let suffix = newVersion;
+
+    // preserve workspace protocol prefix,
+    // tools that use this replace with a real version during the publish the process
+    if (existingVersion.startsWith('workspace:')) {
+      prefix = 'workspace:';
+      range = existingVersion.split(':')[1];
```

Comment:
This is very very slightly off. Since we are only checking that it starts with `workspace:`, we need to slice the beginning instead of split on `:` (otherwise, any values that include multiple `:` would be mangled incorrectly -- I realize this may not actually be a real thing, but there is no reason to diverge between our guard and the final value).

Updated to `existingVersion.slice(prefix.length)` instead.